README.md

```diff
@@ -8,41 +8,54 @@
 
 <img src="https://www.haskell.org/cabal/images/Cabal-light.png" align="right">
 
-This Cabal Git repository contains the following packages:
+This Cabal Git repository contains the following main packages:
 
  * [Cabal](Cabal/README.md): the Cabal library package ([license](Cabal/LICENSE))
  * [Cabal-syntax](Cabal-syntax/README.md): the `.cabal` file format library ([license](Cabal-syntax/LICENSE))
  * [cabal-install](cabal-install/README.md): the package containing the `cabal` tool ([license](cabal-install/LICENSE))
+ * [cabal-install-solver](cabal-install-solve): the package containing the solver component of the `cabal` tool ([license](cabal-install-solver/LICENSE))
 
 The canonical upstream repository is located at
 https://github.com/haskell/cabal.
 
-Installing cabal-install (by downloading the binary)
-----------------------------------------------------
+Ways To Get cabal-install Binary
```

Comment:
To me, the capitalization looks strange.  I like the previous style better (no capitalization by the first word.
Applies to all headings.

Reply:
Thanks for the feedback! Capitalizing titles is a general practice advised by all major style guides (e.g. AP Stylebook, The Chicago Manual of Style, and the AMA Manual of Style). [Citation: Grammarly](https://www.grammarly.com/blog/capitalization-in-the-titles/). I don't see why readmes shouldn't follow it. I'm open to discussion, though. And happy to hear opinions of others.

As to why `cabal-install` goes non-capitalized, my thought was that it's better to leave it so because it's more recognizable that way. It's a name of a package, and those are case sensitive, so changing capitalization in those names looks incorrect. As well as for all other stylistic choices, I won't die over it and happy to implement any decision provided there's some discussion leading to it.

Reply:
I would side with Andreas on this one. Let’s take the Grammarly article as an example:
- the *title* is “Capitalization in Titles” (so with nouns, etc. capitalised);
- but a section header is styled in the same article as “More examples of capitalization in titles”.

`cabal-install` should stay uncapitalised: it is common practice in — say — manpages (even after full stops).

Whichever decision we end up taking, as long as we are consistent it is fine. Even better for contributors if we specify “write concisely and clearly, if you have doubts about style consult “Xyz style guide”).

Reply:
![image](https://user-images.githubusercontent.com/87714215/185443836-f2ebcc2e-319a-497c-b07f-1b1e497f5034.png)

“cabal-install” can be put between `` in titles too, if you feel the title is more readable.